.github/workflows/solutions-uat.yaml

```
@@ -0,0 +1,159 @@
+        run: |
+          sudo apt update
+          sudo apt install -y age
+          mkdir -p $HOME/.config/sops/age/
+          echo # public key: ${{ secrets.PUBLIC_AGE_KEY }} > $HOME/.config/sops/age/keys.txt
+          echo ${{ secrets.PRIVATE_AGE_KEY }} >> $HOME/.config/sops/age/keys.txt
+          wget https://github.com/mozilla/sops/releases/download/v3.7.1/sops-v3.7.1.linux
+          chmod +x sops-v3.7.1.linux
+          sudo mv sops-v3.7.1.linux /usr/local/bin/sops
+
+      - name: sops decryption
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:57: Double quote to prevent globbing and word splitting

(shellcheck)